### PR TITLE
Onboarding - New design

### PR DESCRIPTION
**Pull request by zilaei**
## Explain changes

Complete rewrite of on boarding.
No other component should have been affected or changed.
## How to test the changes?

Check out the branch and then check out the smooth animations, the crisp colors and the cool layouts.
Was this feature tested in the following environments?

- [x] Building the Application on a iOS device/simulator.
- [x] Building the Application on a Android device/simulator.

### CODE REVIEW

File: source/navigator/AuthStack.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { createStackNavigator } from '@react-navigation/stack';
-import OnboardingScreen from '../screens/OnboardingScreen';
+// import OnboardingScreen from '../screens/OnboardingScreen';
```
Comment (jwinding) [inline source/navigator/AuthStack.js:3]: remove comment

File: source/screens/onboarding/Onboarding.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  const backgroundColor = interpolateColor(x, {
+    inputRange: slides.map((_, i) => i * width),
+    outputRange: slides.map((slide) => slide.color),
+  });
+  const lastScrollPos = width * (slides.length - 2);
+
+  return (
+    <OnboardingContainer>
+      <StatusBar hidden />
+      {/* <Animated.View style={[styles.slider, { backgroundColor }]}> */}
```
Comment (jwinding) [inline source/screens/onboarding/Onboarding.tsx:113]: remove comment

**Review by jwinding — APPROVED** [review]
Tested on Android, works well and looks great (better now that the statusbar is removed). And code looks good to, just two comments that can be removed.

**Comment by helsingborgsstad** [comment]
Task linked: [CU-beurq5 Implementera nya onboardingen?](https://app.clickup.com/t/beurq5)

**Review by jonatanhanson — APPROVED** [review]
Snyggt mannen

**Pull request merged**